src/components/ExportMultipleCampaignDataDialog.tsx

```diff
@@ -0,0 +1,140 @@
+import Button from "@material-ui/core/Button";
+import Dialog from "@material-ui/core/Dialog";
+import DialogActions from "@material-ui/core/DialogActions";
+import DialogContent from "@material-ui/core/DialogContent";
+import DialogContentText from "@material-ui/core/DialogContentText";
+import DialogTitle from "@material-ui/core/DialogTitle";
+import Divider from "@material-ui/core/Divider";
+import Typography from "@material-ui/core/Typography";
+import { useExportCampaignsMutation } from "@spoke/spoke-codegen";
+import React, { useState } from "react";
+
+import { CampaignExportModalContent } from "../containers/AdminCampaignStats/components/CampaignExportModal";
+
+export type CampaignDetailsForExport = {
+  id: string;
+  title: string;
+};
+interface Props {
```

Comment:
required: export props, and name `ExportMultipleCampaignDataDialogProps`, more details in https://github.com/politics-rewired/Spoke/blob/main/conventions.md